library/core/src/lib.rs

```diff
@@ -279,6 +279,8 @@ pub mod num;
 pub mod hint;
 pub mod intrinsics;
 pub mod mem;
+#[unstable(feature = "profiling_marker_api", issue = "none")]
+pub mod profiling;
```

Comment:
I think you have to add something to `std/src/lib.rs` to get the API re-exported in `std` as well - that usually documents better. Unless this intentionally isn't done?

Reply:
> If they only ever appear as `core` then making them also appear in `std` could just be confusing.

But this is already the state for all debug info, no? If you call a `core` method that happens to be re-exposed in `std`, you'd still see `core` in your debugger.

> I was unsure about making it an exposed public module but functionally empty. Should that be deferred until it has some actual callable contents?

Unsure too, in the end, this is probably libs domain (?), I'll ping one of them for guidance once the MCP finishes.